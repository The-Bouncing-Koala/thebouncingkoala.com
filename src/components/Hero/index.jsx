/* eslint-disable no-unused-vars */
import React from 'react'
import PropTypes from 'prop-types'
import Logo from '@images/tailwindcss.png'
import './index.pcss'

const Hero = (props) => {
  const { title, image, imageAlt } = props

  return (
    <div className="hero">
      <img src={image} alt={imageAlt} />
      <img src={Logo} alt="Tailwindcss Logo" />
      <h1 className="hero__title">{title}</h1>
    </div>
  )
}

Hero.propTypes = {
  title: PropTypes.string,
  image: PropTypes.string,
  imageAlt: PropTypes.string
}

export default Hero
