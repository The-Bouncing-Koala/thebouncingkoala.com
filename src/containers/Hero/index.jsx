/* eslint-disable no-unused-vars */
import React from 'react'
import PropTypes from 'prop-types'
import Logo from '@images/tailwindcss.png'
import './index.pcss'

const Hero = ({ title, image, imageAlt }) => {
  return (
    <div className="Hero">
      <img src={image} alt={imageAlt} />
      <img src={Logo} alt="Tailwindcss Logo" />
      <h1 className="Hero__Title">{title}</h1>
    </div>
  )
}

Hero.propTypes = {
  title: PropTypes.string,
  image: PropTypes.string,
  imageAlt: PropTypes.string
}

export default Hero
