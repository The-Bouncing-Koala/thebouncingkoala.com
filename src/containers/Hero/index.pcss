.hero {
  @apply w-screen h-screen bg-indigo-900 flex flex-col justify-center items-center;

  &__title {
    @apply text-white text-5xl font-bold;
  }
}
