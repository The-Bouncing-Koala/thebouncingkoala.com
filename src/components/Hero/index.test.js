import React from 'react'
import Hero from './index'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'

describe('<Hero />', () => {
  test('Mount component', () => {
    const hero = mount(<Hero />)
    expect(hero).toHaveLength(1)
  })
  test('Check the UI', () => {
    const hero = create(<Hero />)
    expect(hero.toJSON()).toMatchSnapshot()
  })
})
