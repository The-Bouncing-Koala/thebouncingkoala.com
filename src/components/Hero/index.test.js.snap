// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero /> Check the UI 1`] = `
<div
  className="hero"
>
  <img />
  <img
    alt="Tailwindcss Logo"
    src="test-file-stub"
  />
  <h1
    className="hero__title"
  />
</div>
`;
